### PR TITLE
Use eslint-config-rainbow

### PR DESCRIPTION
**Pull request by bcomnes**
This converts over to the new rainbow sharable eslint config.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -19,10 +19,8 @@ const globalVars = parse(babelParse(data, { sourceType: 'module' }))
   );
 
 module.exports = {
-  extends: 'satya164',
-  plugins: ['jest'],
+  extends: 'rainbow',
   settings: {
-    'react': { version: '16' },
     'import/resolver': {
```
Comment (bcomnes) [inline .eslintrc.js:24]: This didn't work when I had in the shared config, plus its fairly project specific.

**Review by skylarbarrera — APPROVED** [review]
🔥

File: .eslintrc.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/* eslint-disable sort-keys */
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```
Comment (bcomnes) [inline .eslintrc.js:1]: Sort keys wasn't a huge problem in this repo, but it is in the other repos that don't have it. This variation of the rule lets me --fix the sorting stuff, way easier.  

Happy to revert if people are don't on a custom rule though.

Comment (bcomnes) [reply .eslintrc.js:1]: Specifically interested in @brunobar79's opinion here.

**Pull request merged**